_data/kernels.json

```diff
@@ -110,6 +110,8 @@
         [ "--module", "CONFIG_IP_NF_TARGET_REDIRECT" ],
         [ "--module", "CONFIG_NETFILTER_XT_MATCH_COMMENT" ],
 
+        [ "--module", "CONFIG_NETFILTER_XT_MATCH_STATISTIC" ],
```

Comment:
Note that's not the only config kube-proxy will complain about if it's missing. It will complain about all of those:
https://github.com/cilium/packer-ci-build/blob/692e05bcc6a9c05ecbe85b2824e24f76b50e0bdc/provision/ubuntu/kernel-next.sh#L101-L106

Reply:
Thanks!